WordPress/openverse
# Fix logo height

## Fixes



Fixes #476 by @fcoveram.

## Description



Fixes the logo by reducing its size and making its size universal. Also removes "baked in" padding to the logo loader; this instead comes from the context of the loader. 

The Openverse logo should be the same size everywhere, now.

This PR also removes the global `maxDiffPixelRatio` from our Playwright tests. These tests are fortunately much more stable these days, and removing this gives us better assurances we are capturing changes to our front-end. 

For example, with that setting enabled, our tests missed the fact that the "Terms" link was added to the header! Some snapshot changes only reflect the change in this link and not a change to the logo.

## Testing Instructions


Run the PR locally with `just frontend`. The Openverse logo should be the same size everywhere, now. Click through the site and confirm. Tests should pass too.

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
